### DISCUSSION TITLE
Ratio and fit ignored with nuxt/image

### DISCUSSION THREAD

**Discussion by learntheropes** [vercel/community#2164] [Bug Report]
Vercel does not respect the image ratio of `nuxt/image`.

I have images I want to show in square format.

They show correctly with `nuxt generate` locally:

![locally](https://nostr.build/i/nostr.build_84ac42962adbd091772f9ac3ad8749cecd96aaf2bbddda2238057ab4dbf9804a.png)

On Vercel, they only follow the 128px width rule, but they don't follow the `fit: 'conver'` and `height: '128'` rules.

![vercel](https://nostr.build/i/nostr.build_7584d08251124f2c7ee697ec781659bc9ab60157f10bea8cf37ca9f5746e6c05.png)

Link to one of the picture: https://booking.learntheropes.xyz/_vercel/image?url=/py-998154074.jpg&w=128&q=70

- nuxt.config.js
```js
  image: {
    screens: {
      galleryPreview: 128,
      preview: {
        modifiers: {
          format: 'webp',
          fit: 'cover',
          width: 128,
          height: 128,
          quality: 70
        }
      },
    }
  }
```

- my-component.vue
- ```vue
    <div
      v-if="gallery.length"
      v-for="image, index in gallery"
      :key="image"
      class="column is-narrow"
      @click.native="openModal(gallery, index)"
    >
      <figure
        class="image ltr-rounded is-128x128"
      >
        <NuxtImg
          preset="preview"
          :src="'/'+image"
          :alt="image"
        />
      </figure>  
    </div>
- ```

**Comment by angelorc** [comment]
The same issue occurs here! Locally, I can assign different sizes (e.g., 48px, 480px, 1200px), but when I deploy on Vercel, I only get 320px
